modal-go/cls.go

```diff
@@ -195,7 +195,7 @@ func encodeParameter(paramSpec *pb.ClassParameterSpec, value any) (*pb.ClassPara
 		paramValue.SetBytesValue(bytesValue)
 
 	default:
-		return nil, fmt.Errorf("unsupported parameter type: %w", paramType)
+		return nil, fmt.Errorf("unsupported parameter type: %v", paramType)
```

Comment:
This isn't an error so can't take `%w`. Linter will catch this after I install it.